Source/Scene/Batched3DModel3DTileContentProvider.js

```diff
@@ -121,7 +124,8 @@ define([
     Batched3DModel3DTileContentProvider.prototype.request = function() {
         var that = this;
 
-        var promise = RequestScheduler.throttleRequest(this._url, loadArrayBuffer);
+        var distance = this._tile.distanceToCamera;
+        var promise = RequestScheduler.throttleRequest(this._url, loadArrayBuffer, RequestType.TILES3D, distance);
```

Comment:
As mentioned above, we'll want to thread this through to `Model` whenever it is embedded in a tile.